nuntium/templates/nuntium/profiles/messages_per_instance.html

```diff
@@ -152,4 +157,62 @@ <h4 class="modal-title">{% trans "Delete this message?"%}</h4>
     </div>
   </div>
 </div>
+
+<div class="modal fade" id="resend" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+  <form action="{% url 'messages_per_writeitinstance' %}" method="post">{% csrf_token %}
+    <div class="modal-dialog">
+      <div class="modal-content">
+      <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title">Resend this message</h4>
+        </div>
+        <div class="modal-body">
+          <p>Who would you like to resend this message to?</p>
+          <div class="contact-list"></div>
+          <input type="hidden" name="message" value="" id="message_id"/>
+        </div>
+        <div class="modal-footer">
+          <button type="submit" class="btn btn-primary">Resend</button>
+        </div>
+      </div>
+    </div>
+  </form>
+</div>
+
+<script>
+
+  $(document).ready(function() {
+    $('.resend').click(function() {
+      
+      let contacts = $('input', $(this).parent())
+      $('#resend .contact-list').html('');
+      
+      for(i = 0; i < contacts.length; i++) {
+        console.log();
+        let val = $(contacts[i]).val().split('-')[0];
+        let name = $(contacts[i]).val().split('-')[1];
```

Comment:
what if there's a - in the name?

Reply:
The point was that encoding and decoding multiple values is error-prone when there's probably a better way, like using a different data attribute for id and name. Or embedding json in a data attribute of the button.

at https://github.com/OpenUpSA/writeinpublic/pull/30/files/39c83ae73a403f35aea4809d149f845d8220cbae#diff-2d6c177afc23b26ad1418195b6f596c9d5f764953afc88a211f94cab8f064e74R128
```html
<button class="btn btn-default btn-sm resend" data-message="{{message.pk}}" data-contacts="[{&quot;id&quot;: 1, &quot;name&quot;: &quot;Benji&quot;}, {&quot;id&quot;: 2, &quot;name&quot;: &quot;Finley&quot;}]">
```